> HI @AClark-WHONET,
> 
> Automatic Validation ended with:
> 
> > ##[error] (Line: 34, Col: 73, Idx: 1900) - (Line: 34, Col: 73, Idx: 1900): While scanning a plain scalar value, found invalid mapping.
> 
> I'm not a YAML expert, but I believe the colon might need to be escaped.

Thanks Stephen.

--- a/manifests/w/WHONET/WHONET/24.9.25/WHONET.WHONET.locale.en-US.yaml
+++ b/manifests/w/WHONET/WHONET/24.9.25/WHONET.WHONET.locale.en-US.yaml
@@ -31,7 +31,7 @@ Tags:
 - Microbiology
 - Resistance
 - WHO
-ReleaseNotes: Corrects a problem handling breakpoints for serovar groups\: ECO, HIN, NME.
+ReleaseNotes: "Corrects a problem handling breakpoints for serovar groups: ECO, HIN, NME."
 ReleaseNotesUrl: https://whonet.org/WHONET-ReleaseNotes.html
 ManifestType: defaultLocale
 ManifestVersion: 1.6.0

